Tween/AnimationCollection.cs

```diff
@@ -11,6 +11,27 @@ namespace DUCK.Tween
 	/// </summary>
 	public abstract class AnimationCollection : AbstractAnimation, IAnimationPlaybackControl
 	{
+		public IAnimationDriver AnimationDriver
+		{
+			get
+			{
+				return animationDriver ?? (animationDriver = TimedAnimation.DefaultDriver ?? DefaultAnimationDriver.Instance);
```

Comment:
Since they all have the same get, I wonder if an interface on each is still the right choice for this rather than inheriting

Reply:
Set is different. Get is similar. I think it's fine here.

I've tried several different approaches and by far this is the cleanest and required smallest changes.

Reply:
I think interface is the correct solution here. I'd even go as far as suggesting another interface just for the animation driver, since the driver isn't really part of the playback mechanism. It's quite a common mistake to dump everything into 1 interface usually because it already exists in all the places you want to put the new behaviour, 

I think it's fine for now, but I can see it growing if we need to add more behaviour to the same set of classes. Ideally they should be split by their functionality.